template/src/ExamplePanel.tsx

```diff
@@ -16,7 +16,7 @@ function ExamplePanel({ context }: { context: PanelExtensionContext }): JSX.Elem
     //
     // Without a render handler your panel will never receive updates.
     //
-    // The render handler could be invoked as often as 60hz during playback if fields are changing often.
+    // The render handler could be invoked as often as 360hz during playback if fields are changing often.
```

Comment:
What would cause this to be invoked at 360hz? I thought we limit this within Studio message pipeline to the app's settings of message rate (which is max 60hz).